### PR TITLE
Update stalebot.yml

### PR DESCRIPTION
**Pull request by jonjanego**
adding more operations per run, changing the time before close to 30 days, fixing some comments

### CODE REVIEW

**Review by jurre — APPROVED** [review]
🤖

File: .github/workflows/stalebot.yml [hunk 1]
```diff
@@ -20,7 +20,8 @@ jobs:
         stale-pr-label: "Stale"
         exempt-pr-labels: "Keep" # a "Keep" label will keep the PR from being closed as stale
         days-before-pr-stale: 730 # when the PR is considered stale
-        days-before-pr-close: 60 # when the PR is closed by the bot,
-        days-before-issue-stale: 730 # prevents issues from being tagged by the bot
-        days-before-issue-close: 60 # prevents issues from being closed by the bot
+        days-before-pr-close: 30 # when the PR is closed by the bot
```
Comment (abdulapopoola) [inline .github/workflows/stalebot.yml:23]: can we shorten this to 1 week or lesser please?

Comment (jonjanego) [reply .github/workflows/stalebot.yml]: done

**Review by abdulapopoola — APPROVED** [review]
